programs/util.h

```diff
@@ -39,7 +39,7 @@ extern "C" {
 #endif
 #include <time.h>         /* clock_t, clock, CLOCKS_PER_SEC, nanosleep */
 #include "mem.h"          /* U32, U64 */
-
+#include "fileio.h"
```

Comment:
In term of dependency, I don't like the idea to have `util.h` depending on `fileio.h`,
keeping in mind that `fileio.c` depends on `util.h`.

Dependency should be a clean tree, with no circular relations.

Reply:
Addressed in latest commit.